ci/ansible/roles/centos/vars/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+
```

Comment:
Will it be just empty file? If this file is not required then can we get rid of it and add it later when there is its use?